### PR TITLE
Show page number

### PR DESCRIPTION
**Pull request by karszawa**
Thank you for the excellent tool!
I used this tool for the presentation of my graduation thesis.

I needed a page number notation to use this tool in my situation.
Page numbers are useful to address a specific slide in QA session.

I implemented this function as an option.
So, existing slides will not be affected by this change.

### CODE REVIEW

**Comment by hiroppy** [comment]
Thank you for your contribution 🎉  
Currently, Snapshot tests fail so please execute `npm test -- -u`.

File: README.md [hunk 1]
```diff
@@ -160,6 +160,7 @@ slide:
   theme: nebula
   sidebar: true
   targetBlank: true
+  pageNumbber: false
```
Comment (hiroppy) [inline README.md]: typo

**Comment by hiroppy** [comment]
Oops, `__tests__/src/webpack/webpack.prod.config.js` fails. I'll check it.

**Comment by hiroppy** [comment]
@karszawa I fixed this test so please rebase from master 🙏

**Comment by karszawa** [comment]
@hiroppy Rebased. Thanks!

**Comment by hiroppy** [comment]
I published v0.10.0🎉 see https://github.com/hiroppy/fusuma/releases/tag/v0.10.0

**Pull request merged**